ext/redhat/razor-server.env

```diff
@@ -7,4 +7,5 @@ JBOSS_CONFIG=standalone.xml
 JBOSS_MODULES_SYSTEM_PKGS=org.jboss.byteman
 LAUNCH_JBOSS_IN_BACKGROUND=true
 LANG=en_US.UTF-8
-JAVA_OPTS="-Xms128m -Xmx1024m -XX:MaxPermSize=256m -Djava.net.preferIPv4Stack=true"
\ No newline at end of file
+JAVA_OPTS="-Xms128m -Xmx1024m -XX:MaxPermSize=256m -Djava.net.preferIPv4Stack=true"
```

Comment:
This is probably worth keeping in mind. I haven't heard any users bringing up any issues with this (it's been that way since the start). If this comes up as an issue, we can create a ticket and do testing without this argument.